src/main/java/com/faforever/client/chat/Channel.java

```diff
@@ -60,6 +72,12 @@ public ChatUser getUser(String username) {
     return users.get(username);
   }
 
+  public void onChannelTopic(String oldTopic, String topic, String username, long date, boolean changed) {
+    if (this.topic.getValue() == null || changed) {
+      this.topic.setValue(topic);
+    }
```

Comment:
Im Standard-IRC wird man über eine Chat-Zeile über die Änderung des Channel-Topic informiert (inkl. User und Zeitstempel). Es wäre unschön über das gleiche Event mehrfach informiert zu werden. Dies kann ich in der PircBotX Implementierung nicht ausschließen, da ich hier die Event-Mechanismen nicht kenne. Z.B. bei einem Reconnect würde ich mit einem neuen ChannelTopic-Event rechnen obwohl sich nichts geändert hat.
So gehe ich auf Nummer sicher.